pkgs/tools/networking/clarissa/default.nix

```
@@ -0,0 +1,33 @@
+{ stdenv, fetchFromGitLab, pkgs, libpcap, perl }:
+
+stdenv.mkDerivation rec {
+
+  version = "0.6";
+  name = "clarissa-${version}";
+
+  src = fetchFromGitLab {
+    owner = "evils";
+    repo = "clarissa";
+    rev = "v${version}";
+    sha256 = "0xm9v9h4v7s2rdwzfjnmlfas18x9qirrcn01qvclca63pypz3xdk";
+  };
+
+  buildInputs = with pkgs; [ libpcap perl ];
+
+  doCheck = true;
+
+  installFlags = [ "DESTDIR=$(out)" "PREFIX=" "SYSDINST=false" ];
```

probably not necessary, but it's "more correct"
```suggestion
  installFlags = [ "DESTDIR=${placeholder "out"}" "PREFIX=" "SYSDINST=false" ];
```